hal/src/stm32f2xx/rtc_hal.c

```diff
@@ -84,21 +101,187 @@ void HAL_RTC_Initialize_UnixTime()
     /* Get calendar_time date struct values */
     RTC_DateStructure.RTC_WeekDay = 6;
     RTC_DateStructure.RTC_Date = 1;
-    RTC_DateStructure.RTC_Month = 0;
+    RTC_DateStructure.RTC_Month = 1;
     RTC_DateStructure.RTC_Year = 0;
 
     setRTCTime(&RTC_TimeStructure, &RTC_DateStructure);
 }
 
+static void hal_rtc_configure()
```

Comment:
Prefer to return something in case of failure to conditionally execute following procedures.